JavaScript/reactjs/demo_app1/src/Form.js

```diff
@@ -1,24 +1,23 @@
 import React, {Component} from 'react'
 
 class Form extends Component {
-  initialState = {
-    Title: '',
-    Imdb: '',
-    
-  }
 
-  state = this.initialState
+    constructor(props){
+        super(props);
+        this.state = { Title: '', Imdb: ''};
+    }
+
+
+    handleChangeTitle = (event) => {
+        this.setState({ Title: event.target.value });
+    };
+
+    handleChangeImdb = (event) => {
+        this.setState({ Imdb: event.target.value });
+    };
 
-  handleChange = (event) => {
-    const {Title, value} = event.target
-    // const {Imdb} = event.target.Imdb 
-    this.setState({
-      [Title]: value,
-    })
-    
-  }
 
-  submitForm = () => {
+    submitForm = () => {
     
     this.props.handleSubmit(this.state)
     this.setState(this.initialState)
```

Comment:
"this.initialState" is undefiend

Reply:
wo kar di hai update

Reply:
I am now clearing up input form as we submit